pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -37,85 +37,76 @@ const _ = cockpit.gettext;
  * Enables showing shell and ansible script. Shell one is mandatory and ansible one can be omitted.
  *
  */
-class ModificationsExportDialog extends React.Component {
-    constructor(props) {
-        super(props);
+export const ModificationsExportDialog = ({ show, onClose, shell, ansible }) => {
+    const [active_tab, setActiveTab] = React.useState("shell");
+    const [copied, setCopied] = React.useState(false);
+    let timeoutId = null;
 
-        this.timeoutId = null;
-        this.state = {
-            active_tab: "shell",
-            copied: false
-        };
-
-        this.handleSelect = this.handleSelect.bind(this);
-        this.copyToClipboard = this.copyToClipboard.bind(this);
-    }
-
-    handleSelect(event, active_tab) {
-        this.setState({ active_tab, copied: false });
-        if (this.timeoutId !== null) {
-            clearTimeout(this.timeoutId);
-            this.timeoutId = null;
+    const handleSelect = (_event, active_tab) => {
+        setCopied(true);
+        setActiveTab(active_tab);
+        if (timeoutId !== null) {
+            clearTimeout(timeoutId);
+            timeoutId = null;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19370-20230921-132212-9000f540-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)